datasets/europa_eac_tm/europa_eac_tm.py

```diff
@@ -0,0 +1,225 @@
+            **kwargs: keyword arguments forwarded to super.
+        """
+        name = f"{language_pair[0]}2{language_pair[1]}"
+        description = f"Translation dataset from {language_pair[0]} to {language_pair[1]}"
+        super(EuropaEacTMConfig, self).__init__(
+            name=name,
+            description=description,
+            version=datasets.Version("1.0.0", ""),
+            **kwargs,
+        )
+        source, target = language_pair
+        assert source != target, "Source and target languages must be different}"
+        assert source == "en", f"Source language muste be 'en', got {source}"
+        assert (
+            target in _TARGET_LANGUAGES
+        ), f"Source language {target} is not supported. Must be one of : {_TARGET_LANGUAGES}"
+        self.language_pair = language_pair
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class EuropaEacTM(datasets.GeneratorBasedBuilder):
+    """European Commission Joint Research Center's EAC Translation Memory"""
+
+    VERSION = datasets.Version("1.0.0")
+    FORM_SENTENCE_TYPE = "form_data"
+    REFERENCE_SENTENCE_TYPE = "sentence_data"
+
+    BUILDER_CONFIGS = [EuropaEacTMConfig(language_pair=("en", target)) for target in _TARGET_LANGUAGES]
+
+    def _info(self):
+        source, target = self.config.language_pair
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "translation": datasets.features.Translation(languages=self.config.language_pair),
+                    "sentence_type": datasets.features.ClassLabel(
+                        names=[self.FORM_SENTENCE_TYPE, self.REFERENCE_SENTENCE_TYPE]
+                    ),
+                }
+            ),
+            supervised_keys=(source, target),
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        dl_dir = dl_manager.download_and_extract(_DATA_URL)
+        form_data_file = os.path.join(dl_dir, "EAC_FORMS.tmx")
+        reference_data_file = os.path.join(dl_dir, "EAC_REFRENCE_DATA.tmx")
+        source, target = self.config.language_pair
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "form_data_file": form_data_file,
+                    "reference_data_file": reference_data_file,
+                    "source_language": source,
+                    "target_language": target,
+                },
+            ),
+        ]
+
+    def _generate_examples(
+        self,
+        form_data_file,
+        reference_data_file,
+        source_language,
+        target_language,
+    ):
+
+        with open(form_data_file, encoding="utf-8") as f_forms:
+            forms_soup = BeautifulSoup(f_forms.read(), "xml")  # Needs bs4 and lxml as dependencies
```

Comment:
It's not possible to use xml.etree.ElementTree here ?
It would remove the dependencies on bs4 and lxml